field/field.go

```diff
@@ -303,3 +302,20 @@ func extendedEuclideanAlgo(r, q, rInv, qInv *big.Int) {
 	}
 	qInv.Neg(qInv)
 }
+
+//HexToMont takes an element written in hex, and returns it in Montgomery form
+//Useful for hard-coding in implementation field elements from standards documents
+func HexToMont(hex string, fieldModulus *big.Int) big.Int {
+
+	var i big.Int
+	i.SetString(hex, 16)
+	IntToMont(&i, fieldModulus)
+
+	return i
+}
+
+func IntToMont(i *big.Int, fieldModulus *big.Int) {
+	nbWords := (fieldModulus.BitLen()-1)/64 + 1
```

Comment:
does this work on 32bit architecture? 

Can we define these methods on the Field object to ensure we use the same R?